src/navigation/appNavigation.js

```diff
@@ -556,6 +558,7 @@ const AppFlowNavigation = createStackNavigator(
     [ENS_MIGRATION_FLOW]: ensMigrationFlow,
     [ADD_CASH]: addCashFlow,
     [WEB_VIEW]: WebViewScreen,
+    [TERMS_AND_PRIVACY]: TermsAndPrivacyScreen,
```

Comment:
Could we change this to LEGAL_MODAL or something similar for consistency with the rest of the naming please?